app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/AddEvent.kt

```diff
@@ -20,9 +20,9 @@ class AddEvent : Activity(), AdapterView.OnItemSelectedListener {
     // creating empty ArrayList and setting default values
     var period = arrayListOf<String>()
     var periodSelected = 0
-    val days = arrayListOf<String>()
+    private val days = arrayListOf<String>()
     var timeSpecification = -1
-    val event = Event(isDateSet = false, isDaysSet = false, days = "", date = "", time = "", period = "AM", timeSpecification = "")
+    private val event = Event(isDateSet = false, isDaysSet = false, days = "", date = "", time = "", period = "AM", timeSpecification = "")
```

Comment:
These vars should have been private here there was no reason not to declare these it is good to get into the habit of declaring this while creating variables throughout your code.

Reply:
This appeared in a previous review as well.  When that occurs, first in wins.